### ISSUE TITLE
Screwdriver not working as intended

### ISSUE THREAD

**Issue by CobaltWolf** [CobaltWolf/Surface-Experiment-Pack#2]

**Comment by cake-pie** [comment]
It's functionally identical and interchangeable with KIS electric screwdriver at this point.
This is a limitation of KIS ihsoft/KIS#126 and will need new feature from KIS side to enable other mods tools to be "mod-specific".

**Comment by CobaltWolf** [comment]
Yup, I figured that out some time ago. Nothing we can do I think.
